### DISCUSSION TITLE
Add FreeBSD support

### DISCUSSION THREAD

**Discussion by drupaladmin** [Releem/releem-agent#206] [Feature Requests]
Releem Agent cound be installed on Linux and as a Docker container
 
The issue on how to run Releem Agent manually on FreeBSD https://github.com/Releem/mysqlconfigurer/issues/203

Please add FreeBSD support

**Comment by drupaladmin** [comment]
How to run Releem Agent on the FreeBSD manually - https://github.com/Releem/mysqlconfigurer/issues/203

**Comment by manswiss** [comment]
See https://github.com/Releem/mysqlconfigurer/issues/203#issuecomment-2557163367 with a nearly working version. It gives an idea how it can be done. I think the effort is not very big to bring it work.
